src/main/java/mate/controller/driver/DeleteDriverController.java

```diff
@@ -0,0 +1,25 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.DriverService;
+
+@WebServlet("/drivers/delete")
+public class DeleteDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService = (DriverService)
+            injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long id = Long.valueOf(req.getParameter("id"));
+        driverService.delete(id);
```

Comment:
```suggestion
        Long driverId = Long.valueOf(req.getParameter("id"));
        driverService.delete(driverId );
```

Reply:
I don’t think it’s necessary. The controller is called as `DeleteDriverController` so it is clear from the name of the controller that I am working here with the entity of the driver